okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -136,6 +138,23 @@ public Level getLevel() {
     return level;
   }
 
+  /**
+   *
+   * @return check mutipart logging is enabled or not
+   */
+  public boolean isShowMultipartLog() {
+    return showMultipartLog;
+  }
+
+  /**
+   *
+   * @param showMultipartLogging set to false to display multipart length in integer instead of non-plain text (by default true)]
```

Comment:
Can you clean up these javadoc, put most of the comment to the body and less in the param and return.

Reply:
sure..